src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http3WebtransportManager.cs

```diff
@@ -41,17 +41,16 @@ public bool addSession(QuicStream connectStream, Http3WebtransportSession webtra
         public bool AcceptServerStream(QuicStream stream, long sessionId)
         {
             Http3WebtransportSession? session;
-            Console.WriteLine(" SPEEER nuuuuu " + sessionId);
             _sessions.TryGetValue(sessionId, out session);
             // if no session with that id exists throw exception
             // https://datatracker.ietf.org/doc/html/draft-ietf-webtrans-http3#section-4
             if (session == null)
             {
+                // TODO: Mana
                 Console.WriteLine("Throw error");
                 throw HttpProtocolException.CreateHttp3ConnectionException(Http3ErrorCode.WebtransportBufferedStreamRejected);
```

Comment:
You're not suppose to throw this error but rather response to server with it. Look into https://docs.microsoft.com/en-us/dotnet/api/system.net.quic.quicstream.abort?view=net-7.0#system-net-quic-quicstream-abort(system-net-quic-quicabortdirection-system-int64)